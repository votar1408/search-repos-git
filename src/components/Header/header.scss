.header-root {
    position: fixed;
    left: 0;
    top: 0;
    padding: 20px;
    width: 100%;
    height: 60px;
    color: white;
    font-weight: bold;
    background-color: #222;
    border-bottom: 1px solid white;
}
