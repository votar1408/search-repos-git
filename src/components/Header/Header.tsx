import './header.scss';

import React, {FC} from 'react';

import {texts} from '../../consts';

export const Header: FC = () => {
    return (
        <header className="header-root">
            <div>
                <span>{texts.appName}</span>
            </div>
        </header>
    );
};
