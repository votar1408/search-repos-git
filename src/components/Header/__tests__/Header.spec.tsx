import React from 'react';

import {
    shallow,
    ShallowWrapper
} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';

import {Header} from '../Header';

describe('Header', () => {
    const wrapper: ShallowWrapper = shallow(<Header />);

    it('should render correctly', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });

    it('renders without crashing', () => {
        expect(wrapper).toBeTruthy();
    });

    it('should contains 1 <span/>', () => {
        expect(wrapper.find('span').length).toEqual(1);
    });
});
