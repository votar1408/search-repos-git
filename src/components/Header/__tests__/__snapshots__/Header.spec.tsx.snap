// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<header
  className="header-root"
>
  <div>
    <span>
      Search of Repositories in GitHub
    </span>
  </div>
</header>
`;
